spec/acceptance/suites/default/00_setup_389ds_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'spec_helper_acceptance'
+
+test_name 'Set up ds389 server '
+
+describe 'Setup 389ds server for freeradius' do
+  #stunnel just needs to be set, it does not effect this test
+  stunnel_setting = true
+  ldap_server = only_host_with_role(hosts,'389ds')
+  ldap_server_fqdn = fact_on(ldap_server, 'fqdn')
+
+  _domains = fact_on(ldap_server, 'domain').split('.')
+  _domains.map! { |d|
+    "dc=#{d}"
+  }
+  base_dn = _domains.join(',')
+  common_hieradata =  File.read(File.expand_path('files/common_hieradata.yaml.erb', File.dirname(__FILE__)))
+
+  context 'setup 389ds ldap server ' do
+    let(:root_pw)          { 'suP3rP@ssw0r!' }
+    let(:ldap_type)        { '389ds' }
+    let(:server_hieradata) { File.read(File.expand_path("files/#{ldap_type}/server_hieradata.yaml.erb", File.dirname(__FILE__))) }
+    let(:hieradata)        { "#{common_hieradata}" + "\n#{server_hieradata}" }
+    let(:add_testuser)     { File.read(File.expand_path("files/#{ldap_type}/add_testuser.erb", File.dirname(__FILE__))) }
+    let(:ds_root_name)     { 'accounts' }
+
+    it 'should install, 389ds accounts instance' do
+
+      server_manifest = <<-EOM
+        include 'simp_options'
+        include 'simp_ds389::instances::accounts'
+        include 'simp_openldap::client'
```

Comment:
why is this needed?

Reply:
I'm actually surprised that you need the `simp_openldap::client` class at all. What did this pull in that was required?